### PR TITLE
Separate create  / delete config of oauth providers

### PR DESCRIPTION
**Pull request by tung2744**
ref DEV-1313

### CODE REVIEW

**Comment by tung2744** [comment]
@louischan-oursky 
https://github.com/authgear/authgear-server/pull/4234
And this pr is ready for review.
I will open a new issue for the ui updates for the delete button.

File: go.mod [hunk 1]
```diff
@@ -216,3 +216,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.3.0 // indirect
 )
+
+replace github.com/authgear/oauthrelyingparty => github.com/tung2744/authgear-oauthrelyingparty v1.2.0-3684db06
```
Comment (louischan-oursky) [inline go.mod:220]: v1.3.0 was published. Please use it instead.

Comment (tung2744) [reply go.mod]: Switched to v1.3.0, thanks!

**Comment by louischan-oursky** [comment]
Please rebase.

**Comment by tung2744** [comment]
Rebased

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**